specs/electra/beacon-chain.md

```diff
@@ -423,10 +435,50 @@ class BeaconState(Container):
     # [New in Electra:EIP7251]
     pending_partial_withdrawals: List[PendingPartialWithdrawal, PENDING_PARTIAL_WITHDRAWALS_LIMIT]
     pending_consolidations: List[PendingConsolidation, PENDING_CONSOLIDATIONS_LIMIT]  # [New in Electra:EIP7251]
+    # [New in Electra: compute blob gas in CL]
+    excess_blob_gas: uint64
+    base_fee_per_blob_gas: uint64
```

Comment:
do we need to keep this parameter to be in the state as it can be easily computed given the state?